### PR TITLE
Added a fresh coat of paint to streaks.js

### PR DESCRIPTION
**Pull request by j-cordz**
I added a new style to the streaks page so the title is responsive like on the main Scrapbook page, fixed a bit the formatting and added a new value to the link's colors.

### CODE REVIEW

File: pages/streaks.js [hunk 1]
```diff
@@ -24,17 +24,14 @@ const StreaksPage = ({ users }) => {
           font-size: 20px !important;
           line-height: 1.625;
         }
-
         h1 {
           color: var(--colors-orange);
           font-family: var(--fonts-display);
-          font-size: 36px;
-          text-align: center;
```
Comment (sampoder) [inline pages/streaks.js:32]: Looks awesome! Let's keep this line in :D

Comment (j-cordz) [reply pages/streaks.js:32]: Sure! :D

Comment (j-cordz) [reply pages/streaks.js:32]: I added back the line @sampoder, I removed that one by accident, thanks for noticing!

**Comment by sampoder** [comment]
lgtm!

**Comment by j-cordz** [comment]
Yay! 🎉

**Comment by j-cordz** [comment]
This PR had a formatting error I later fixed on #461

**Pull request merged**